app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  before_action :set_company_job
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
you could leverage your associations more by doing the following:

 `@comment = @job.comments.new(comment_params)`

rather than setting the foreign key manually 👍